src/components/Logo.tsx

```diff
@@ -1,4 +1,11 @@
-const Logo = ({ title, classNme }) => {
+import React from 'react';
+
+interface LogoProps {
+  title: string;
+  classNme: string;
+}
+
+const Logo: React.FC<LogoProps> = ({ title, classNme }) => {
```

Comment:
The same goes here `ComponentType<'span'>` will get you all the props that you need for the parent node then you can extends it with `title`